components/brave_wallet/browser/swap_service.cc

```diff
@@ -376,159 +349,222 @@ GURL SwapService::GetZeroExTransactionURL(
 }
 
 // static
-GURL SwapService::GetJupiterQuoteURL(const mojom::SwapQuoteParams& params) {
+GURL SwapService::GetJupiterQuoteURL(
+    const mojom::SwapQuoteParams& params,
+    const std::optional<std::string> fee_param) {
   std::string spec = base::StringPrintf(
-      "%sv6/quote", GetBaseSwapURL(params.from_chain_id).c_str());
+      "%s/v6/quote", GetBaseSwapURL(params.from_chain_id).c_str());
   GURL url(spec);
-  url = AppendJupiterQuoteParams(url, params);
+  url = AppendJupiterQuoteParams(url, params, fee_param);
 
   return url;
 }
 
 // static
 GURL SwapService::GetJupiterTransactionURL(const std::string& chain_id) {
   std::string spec =
-      base::StringPrintf("%sv6/swap", GetBaseSwapURL(chain_id).c_str());
+      base::StringPrintf("%s/v6/swap", GetBaseSwapURL(chain_id).c_str());
+  GURL url(spec);
+  return url;
+}
+
+// static
+GURL SwapService::GetLiFiQuoteURL() {
+  std::string spec =
+      base::StringPrintf("%s/v1/advanced/routes", kLiFiBaseAPIURL);
```

Comment:
why not just use the specially crafted `GURL(kLiFiBaseAPIURL).Resolve("/v1/etc")` ?
This is shorter, safer and also you wouldn't need to remove trailing `/` in urls and they will be correct GURLs

Reply:
Fixed here: 6129ea88c5482f66a823b8b71810e30e3273f0e8